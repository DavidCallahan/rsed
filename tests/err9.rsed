cmd = "./foox"
print $cmd
print shell($cmd)
