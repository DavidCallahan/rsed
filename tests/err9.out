sh: ./foox: No such file or directory
./foox
script 3: error in command: ./foox 
